src/uhttp.c

```diff
@@ -307,9 +309,56 @@ static int process_header_line(const unsigned char* buffer, size_t len, size_t*
     return result;
 }
 
+static void log_text_line(const char* text_line)
+{
+    char* authStart = strstr(text_line, AUTH_HEADER);
+    char* proxyAuthStart = strstr(text_line, PROXY_AUTH_HEADER);
+    char* authEol = NULL;
+    char* proxyAuthEol = NULL;
+
+    if (authStart != NULL)
+    {
+        authEol = strstr((authStart + 1), HTTP_CRLF_VALUE);
+    }
+    if (proxyAuthStart != NULL)
+    {
+        proxyAuthEol = strstr((proxyAuthStart + 1), HTTP_CRLF_VALUE);
+    }
+    if (authStart == NULL && proxyAuthStart == NULL)
+    {
+        LOG(AZ_LOG_TRACE, LOG_LINE, "%s", text_line);
+    }
+    else if (proxyAuthStart == NULL)
+    {
+        if (authEol != NULL)
+        {                                                                 
+            LOG(AZ_LOG_TRACE, LOG_LINE, "%.*s \r\nAuthorization: *** %.*s", (int)(authStart  - text_line), text_line, (int)(strlen(text_line) - (authEol - text_line)), authEol);
```

Comment:
can we get these long log lines shorter with newlines for each of the variables?
```suggestion
            LOG(AZ_LOG_TRACE, LOG_LINE, "%.*s \r\nAuthorization: *** %.*s",
                                                                 (int)(authStart  - text_line), text_line,
                                                                 (int)(strlen(text_line) - (authEol - text_line)), authEol);
```